pom.xml

```diff
@@ -59,63 +59,65 @@
-		<appointmentschedulinguiVersion>1.9.0</appointmentschedulinguiVersion>
-		<calculationVersion>1.2</calculationVersion>
-		<coreappsVersion>1.31.0</coreappsVersion>
+		<adminuiVersion>1.6.0</adminuiVersion>
+		<addresshierarchyVersion>2.14.2</addresshierarchyVersion>
+		<allergyuiVersion>1.8.4</allergyuiVersion>
+		<appframeworkVersion>2.16.0</appframeworkVersion>
+		<appuiVersion>1.14.0</appuiVersion>
+		<appointmentschedulingVersion>1.16.0</appointmentschedulingVersion>
+		<appointmentschedulinguiVersion>1.12.0</appointmentschedulinguiVersion>
+		<calculationVersion>1.3.0</calculationVersion>
+		<coreappsVersion>1.34.0</coreappsVersion>
 		<databaseBackupVersion>1.3.0</databaseBackupVersion>
 		<dataEntryStatisticsVersion>1.7.0</dataEntryStatisticsVersion>
-		<dataexchangeVersion>1.3.3</dataexchangeVersion>
-		<dataintegrityVersion>4.4.3</dataintegrityVersion>
-		<emrapiVersion>1.29.0</emrapiVersion>
-		<eventVersion>2.9.0</eventVersion>
-		<fhir2Version>1.2.2</fhir2Version>
+		<dataexchangeVersion>1.3.7</dataexchangeVersion>
+		<dataintegrityVersion>4.4.4-SNAPSHOT</dataintegrityVersion>
+		<emrapiVersion>1.32.0</emrapiVersion>
+		<eventVersion>2.10.0</eventVersion>
+		<fhir2Version>1.3.0</fhir2Version>
 		<formentryappVersion>1.4.2</formentryappVersion>
-		<formfilterVersion>1.0.0</formfilterVersion>
-		<htmlformentryuiVersion>1.10.0</htmlformentryuiVersion>
-		<htmlformentryVersion>4.3.0</htmlformentryVersion>
+		<formfilterVersion>1.3.0</formfilterVersion>
+		<htmlformentryuiVersion>2.0.0</htmlformentryuiVersion>
+		<htmlformentryVersion>4.4.0</htmlformentryVersion>
 		<htmlwidgetsVersion>1.10.0</htmlwidgetsVersion>
-		<idgenVersion>4.5.0</idgenVersion>
+		<idgenVersion>4.7.0</idgenVersion>
 		<jacksonVersion>1.8.3</jacksonVersion>
 		<junitParamsVersion>1.0.4</junitParamsVersion>
-		<legacyuiVersion>1.8.2</legacyuiVersion>
-		<metadatadeployVersion>1.11.0</metadatadeployVersion>
-		<metadatamappingVersion>1.3.4</metadatamappingVersion>
-		<metadatasharingVersion>1.6.0</metadatasharingVersion>
-		<openconceptlabVersion>1.1</openconceptlabVersion>
-		<openwebappsVersion>1.10.0</openwebappsVersion>
-		<patientFlagsVersion>3.0.2</patientFlagsVersion>
-		<providermanagementVersion>2.11.0</providermanagementVersion>
-		<referenceapplicationVersion>2.10.0</referenceapplicationVersion>
-		<referencemetadataVersion>2.10.1</referencemetadataVersion>
-		<registrationappVersion>1.18.0</registrationappVersion>
-		<registrationcoreVersion>1.9.0</registrationcoreVersion>
-		<reportingRestVersion>1.11.0</reportingRestVersion>
+		<legacyuiVersion>1.8.4</legacyuiVersion>
+		<metadatadeployVersion>1.13.0</metadatadeployVersion>
+		<metadatamappingVersion>1.4.0</metadatamappingVersion>
+		<metadatasharingVersion>1.8.0</metadatasharingVersion>
+		<openconceptlabVersion>1.2.9</openconceptlabVersion>
+		<openwebappsVersion>1.13.0</openwebappsVersion>
+		<patientFlagsVersion>3.0.3-SNAPSHOT</patientFlagsVersion>
```

Comment:
@slubwama lets depend on the latest released version of Patient Flags and Data Integrity modules

Reply:
@slubwama please update these snapshots to the latest versions that have been released.